dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
```

Comment:
We should check with @dedemorton how she feels about having this lightweight getting started guide in here, vs just a link to the docs getting started.